src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Specific.cs

```diff
@@ -99,10 +99,10 @@ public void ConnectWithConflictingDirections_Throws_UnauthorizedAccessException(
                     server.ReadMode = PipeTransmissionMode.Message;
                     Assert.Equal(PipeTransmissionMode.Message, server.ReadMode);
 
+                    client.Connect();
```

Comment:
why is this outside of clientTask? does it matter?

Reply:
That is the more likely point of failure and it does not wait, it succeeds or not, so having it out make the error message more clear that if it was in the task.